src/vmm/src/arch/x86_64/mod.rs

```diff
@@ -143,8 +146,16 @@ pub fn configure_system(
         params.hdr.ramdisk_size = u32::try_from(initrd_config.size).unwrap();
     }
 
-    add_e820_entry(&mut params, 0, EBDA_START, E820_RAM)?;
-    add_e820_entry(&mut params, EBDA_START, EBDA_SIZE, E820_RESERVED)?;
+    // We mark first [0x0, EBDA_START) region as usable RAM
+    // and the subsequet [EBDA_START, (EBDA_START + EBDA_SIZE)) and
```

Comment:
we only mark the `[SYSTEM_MEM_START, SYSTEM_MEM_START + SYSTEM_MEM_SIZE]` as reserved (which I guess implicitly also means the EBDA range is marked as reserved, since that's just the first KiB of the 257KiB system range, but the comment is a bit confusing imo)

Reply:
Fixed the comment. Thanks for catching this.